src/V2/ApplicationV2/Models/Subscriptions/SubscriptionMonitoredItem.cs

```diff
@@ -9,13 +9,17 @@ public record SubscriptionMonitoredItem
     {
         public NodeId NodeId { get; set; } = string.Empty;
         public int SamplingInterval { get; set; } = 1000;
-        public int PublishingInterval { get; set; } = 1000;//TODO: remove from here
-        public int HeartbeatInterval { get; set; } = 1000;
+
+        //TODO: Move to subscription - requires schema changes
+        public int PublishingInterval { get; set; } = 1000;
+
+        //TODO: Move to subscription - makes no sense on monitoredItem, requires schema changes
+        public uint KeepAliveCount { get; set; } = 1000; //Replaces heartbeatinterval which does not exist in OPC UA specification
+
         public uint AttributeId { get; set; } = Attributes.Value;
         public MonitoringMode MonitoringMode { get; set; } = MonitoringMode.Reporting;
         public uint QueueSize { get; set; } = 1;
         public bool DiscardOldest { get; set; } = false;
+        public byte Priority { get; set; } = 0;
```

Comment:
Out of curiosity, where do we use this setting. I could not see any reference to it in the PR

Reply:
It was not used previously in V1.
If not specified, the OPC UA stack uses a default of 0.
I think this might be something a developer would like to override from a config point of view.
We made some other previously hardcoded values configurable, so I thought it would be good to expose Priority too.